docker-compose.yml

```diff
@@ -13,13 +13,15 @@ services:
     command: /work/bin/wait_then_run
     build: .
     image: hmtcentral.azurecr.io/contracts:latest
-    links: 
+    links:
       - ganache
       - ipfs
       - hmt
     volumes:
-      - hmt_token_address:/deployed
-  
+      - hmt_token_address:/deployed-hmttoken
```

Comment:
I'd like to avoid double t. `deployed_hmtoken` is prettier

Reply:
ok